README.md

```diff
@@ -203,9 +206,66 @@ If you want to use other connectors, you can:
 
 #### Mount a volume containing the plugins
 
-1. Prepare a PersistentVolume which contains a directory with your plugin(s) and their dependencies and ensure 
-   these files are world-readable (`chmod -R a+r /path/to/your/directory`).
-2. Mount the volume into your Pod at the path `/opt/kafka/plugins/`
+You can create Kubernetes / OpenShift volume which contains additional plugins and modify the Kafka Connect 
+deployment to use this volume. Since distributed Kafka Connect cluster can run on multiple nodes you need to 
+make sure that the volume can be mounted as read only into multiple pods at the same time. Which volume types 
+can be mounted read only on several pods can be found in [Kubernetes documentation][k8srwomany]. Once you have 
+such volume, you can edit the deployment YAML file according to the example bellow:
+
+1. Open the [OpenShift template](kafka-connect/resources/openshift-template.yaml) 
+or [Kubernetes deployment](kafka-connect/resources/kafka-connect.yaml)
+2. add the `volumeMounts` and `volumes` sections in the same way as in the example below
+3. Redeploy Kafka Connect for the changes to take effect
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kafka-connect
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: kafka-connect
+    spec:
+      containers:
+        - name: kafka-connect
+          image: enmasseproject/kafka-connect:latest
+          ports:
+            - name: rest-api
+              containerPort: 8083
+              protocol: TCP
+          env:
+            - name: KAFKA_CONNECT_BOOTSTRAP_SERVERS
+              value: "kafka:9092"
+          livenessProbe:
+            httpGet:
+              path: /
+              port: rest-api
+            initialDelaySeconds: 60
+          volumeMounts:
+            - mountPath: /opt/kafka/plugins
+              name: pluginsvol
+      volumes:
+        - name: pluginsvol
+          glusterfs: 
+            endpoints: glusterfs-cluster
+            path: kube_vol
+            readOnly: true
+```
+
+Alternatively you can distribute your plugins to all your cluster nodes into the same path, make them 
+world-readable (`chmod -R a+r /path/to/your/directory`) and use the hostPath volume to mount them into 
+your Kafka Connect deployment. To do so use the following volume specification with the example above.
+
+```yaml
+volumes:
+  - name: pluginsvol
+    hostPath:
+      path: /path/to/my/plugins
+      type: Directory
```

Comment:
Maybe it's better to use the `hostpath` as the main example for doing that and leaving the GlusterFS as an example of alternatives ?

Reply:
Fixed.